File: FindMyReport/FindMyReport/client/src/components/Patient/Patient.js
import React from "react";

const Patient = ({ patient }) => {
  return (
    <tbody>
    <tr className="text-left px-2">
      <td className="text-center px-2">{patient.id}</td>
      <td className="text-left px-2">{patient.fullName}</td>
      <td className="text-left px-2">{patient.dob}</td>
      <td className="text-left px-2">{patient.address}</td>
      <td className="text-left px-2">{patient.city}</td>
      <td className="text-left px-2">{patient.state}</td>
      <td className="text-left px-2">{patient.zipCode}</td>
      <td className="text-left px-2">{patient.phone}</td>
      <td className="text-left px-2">{patient.race.name}</td>
    </tr>
    </tbody>
  );
};
export default Patient;

File: FindMyReport/FindMyReport/Models/Patient.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Runtime.CompilerServices;
using System.Runtime.InteropServices;

namespace FindMyReport.Models
{
    public class Patient
    {
        public int Id { get; set; }

        [Required]
        [MaxLength(50)]
        public string FirstName { get; set; }

        [Required]
        [MaxLength(50)]
        public string LastName { get; set; }
        [Required]
        public string Address { get; set; }
        [Required]
        public string City { get; set; }
        [Required]
        public string State { get; set; }

        [Required]
        public string ZipCode { get; set; }

        [Required]
        public string Phone { get; set; }
        [Required]
        public DateTime DOB { get; set; }

        public Race Race { get; set; }
        [Required]
        public int RaceId { get; set; }






        public string FullName
        {
            get
            {
                return $"{FirstName} {LastName}";
            }
        }
    }
}